backend/src/main/java/ca/bc/gov/app/entity/client/ClientTypeCodeEntity.java

```diff
@@ -23,6 +23,10 @@
 @With
 public class ClientTypeCodeEntity extends ExpirableBaseEntity {
 
+  public static final String INDIVIDUAL = "I";
```

Comment:
Constants should be added to the `ApplicationConstants.java` file

Reply:
Ok. I can to this. The way I used to do this was directly on the corresponding entity.